Makefile

```diff
@@ -85,35 +102,35 @@ compile-port: $(OUTPUT_DIR)/libp2p_port
 
 
 # Start application with Beacon API.
-start: compile-native compile-port
+start: $(PROTOBUF_EX_FILES) compile-native compile-port
 	iex -S mix phx.server
 
 # Run an interactive terminal with the main supervisor setup.
-iex: compile-native compile-port
+iex: $(PROTOBUF_EX_FILES) compile-native compile-port
 	iex -S mix
 
 # Run an interactive terminal using checkpoint sync.
-checkpoint-sync: compile-native compile-port
+checkpoint-sync: $(PROTOBUF_EX_FILES) compile-native compile-port
```

Comment:
I see `$(PROTOBUF_EX_FILES) compile-native compile-port` repeated across several commands. Can't we just create another command, for example "compile" that aggregates these?

Reply:
Yeah. That's a good idea!